projects/1.18/assets/biomes-o-plenty/biomesoplenty/lang/zh_cn.json

```
@@ -1,90 +1,80 @@
 {
   "advancements.biomesoplenty.root.title": "超多生物群系",
-  "advancements.biomesoplenty.root.description": "探索周围的世界",
-  "advancements.biomesoplenty.all_biomes.title": "漂泊者",
-  "advancements.biomesoplenty.all_biomes.description": "发现超多生物群系所有的生物群系",
+  "advancements.biomesoplenty.root.description": "探索周遭世界",
+  "advancements.biomesoplenty.all_biomes.title": "探索者",
```

```suggestion
  "advancements.biomesoplenty.all_biomes.title": "羁旅者",
```
原文`Wanderer`
我倾向贴合原文，毕竟这不是`Explorer`
如果要浪漫文雅些，羁旅（在外长期漂泊）或许更好。

---

支持